src/status_im/contexts/wallet/send/transaction_progress/view.cljs

```diff
@@ -25,6 +26,31 @@
     (some (fn [[_k v]] (= (:status v) :confirmed)) transaction-details)   :confirmed
     :else                                                                 nil))
 
+(defn show-save-address-bottom-sheet
+  ([] (show-save-address-bottom-sheet :blue))
+  ([_color]
+   (debounce/throttle-and-dispatch [:open-modal :screen/wallet.save-address] 1000)))
```

Comment:
could you elaborate please on why throttle is needed

Reply:
https://github.com/status-im/status-mobile/pull/19079#discussion_r1523223387